### PR TITLE
chore: use concurrency in actions

### PR DESCRIPTION
**Pull request by rokroskar**
addresses #2524

### CODE REVIEW

**Comment by rokroskar** [comment]
This seems to work: https://github.com/SwissDataScienceCenter/renku/actions/runs/2028896758

**Review by ableuler — DISMISSED** [review]

File: .github/workflows/pull-request-test.yml [hunk 1]
```diff
@@ -8,14 +8,11 @@ on:
       - closed
       - edited
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
```
Comment (Panaetius) [inline .github/workflows/pull-request-test.yml:12]:
```suggestion
  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
```
maybe? so it cancels everything on the same PR if it runs on a PR?

Comment (rokroskar) [reply .github/workflows/pull-request-test.yml:12]: ref is the branch, no?

Comment (Panaetius) [reply .github/workflows/pull-request-test.yml:12]: hmm I guess it should. But I see a lot of repos using the above in the wild.

Comment (Panaetius) [reply .github/workflows/pull-request-test.yml:12]: I think if you had multiple PRs to merge the same branch into different branches (like a hotfix branch into master and develop if you follow proper git-flow) and the trigger to run actions is on PR, one PR would cancel the other PR?

Comment (rokroskar) [reply .github/workflows/pull-request-test.yml:12]: gitflow is beyond my ability of comprehension. If we have the PR number though do we need the ref in there at all? This workflow only triggers on PRs

**Review by ableuler — APPROVED** [review]

**Pull request merged**